ui/pages/permissions-connect/connect-page/utils.ts

```diff
@@ -69,3 +72,129 @@ export function getCaip25PermissionsResponse(
     },
   };
 }
+
```

Comment:
Going to move versions of these helpers into the `@metamask/chain-agnostic-permission` package in a subsequent PR